libraries/AP_HAL_ESP32/targets/esp32/esp-idf/sdkconfig.orig

```diff
@@ -0,0 +1,1728 @@
+#
+# Automatically generated file. DO NOT EDIT.
```

Comment:
the .orig file should not be committed

Reply:
files esp32s3empty.h and esp32buzz.h ended up in commit by error. I ws testing build system and forgot to cancel modifications.